lib/data/inspection.dart

```diff
@@ -0,0 +1,19 @@
+class StartTime {
```

Comment:
U can use DateTime to store both start time and selected , and use Timestamp to store time value on firestore